addon/authenticators/jwt.js

```diff
@@ -44,6 +44,15 @@ export default BaseAuthenticator.extend({
   passwordAttributeName: 'password',
 
   /**
+    The name of the key in the response containing the JWT.
+    @property tokenAttributeName
+    @type String
+    @default 'token'
+    @public
+  */
+  tokenAttributeName: 'token',
```

Comment:
I hadn't realized the token attribute name would have to be in sync between the authorizer and the authenticator, good point, if they are different things will fail. 

But I don't like that now they are two separate properties and you *must* update them both and they *must* be the same. It's user error waiting to happen. What do you think about keeping the declaration here in the authenticator and importing its value in the authorizer?

Reply:
That makes sense, but how would we do that?  I've never done something like that.
```
import jwtAuthenticator from 'ember-simple-auth-jwt/authenticators/jwt'
...
const tokenAttributeName = jwtAuthenticator.get('tokenAttributeName')
```
didn't seem to work.

Reply:
Yeah that wouldn't work, it's just importing an un-instantiated class from the Ember object model.
You could mess with it's prototype chain and get the value that way, but that'd be pretty weird to do.

You could use a Mixin to share a single property between the two classes, but then we're breaking up were these configurable values are defined (some in the authenticator, and one in a mixin).

The cleanest solution I can think of would be to move all the configurable values into _config/environment.js_ and have that as the one place where someone using this library can customize it's values. A downside is that is breaks from the ember-simple-auth pattern of customizations happening directly on the classes, and that's not something I'd want to do if we can avoid it.

... let me think about this and I'll get back to you.

Reply:
Ok how about this:

```
  authorize(data = {}, block = () => {}) {
    let dataKeys = Object.keys(data);

    if (!Ember.isEmpty(data) && dataKeys.length === 2) {
      let tokenKey = dataKeys.filter(key => {
        return key !== 'authenticator'
      });
      block('Authorization', `Bearer ${data[tokenKey]}`);
    }
  }
```

This is of course working under the assumption that ember-simple-data does not change or add any more elements to the _session.authenticated_ data.
If that day comes, this will break but we can deal with it then.

Reply:
You can even throw an assert to catch if this assumption fails in the future; the dataKeys length !==2 or if the data[tokenKey] property is empty.

Bonus points for a test case!